Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -185,12 +185,44 @@ void MatchTOF::run(const o2::globaltracking::RecoContainer& inp, unsigned long f
             matchingPair.setFakeMatch();
           }
         }
+      } else {
+        for (auto& matchingPair : mMatchedTracksPairsSec[sec]) {
+          int bct0 = int((matchingPair.getSignal() - 10000) * Geo::BC_TIME_INPS_INV);
+          float tof = matchingPair.getSignal() - bct0 * Geo::BC_TIME_INPS;
+          if (abs(tof - matchingPair.getLTIntegralOut().getTOF(2)) < 600) {
```

Comment:
Do you have any idea about the efficiency and fake positives from such a definition of fakes (from comparison to MC)?
The BC duration (bunch length) is ~500 ps, isn't subtracting 10ns for the time-of-flight (?) and requiring 600 ps coincidence with expected TOF wrt nominal BC center too crude?

Reply:
I didn't check with MC but I did on data. This is an example (unfortunately pt instead of p in the a-axis)
![immagine](https://github.com/AliceO2Group/AliceO2/assets/9963644/24629ca4-96a6-451e-bc60-e7f89e3a1747)



The assumption is that the minimal time of flight is ~12.5 ns so I kept 2.5 ns of margin.
The spread in time of interacting BC is at the level of 200-250 ps -> 2.5 ns is a 10 sigma margin in this respect.
This should safely allow to get the right BC and to maximizie  the dynamic range for slow particles (beta < 1), ~ 20 ns.
Then, I used 600 ns (~3 sigma cut). I agree it is a bit crude but it can be recalculated also a posteriori since all variables needed are written in the tree.
For a better tuning we need some studies, but it is good to start with something which is reliable.

Reply:
This is when requiring both ITS and TPC
![immagine](https://github.com/AliceO2Group/AliceO2/assets/9963644/dfed7e64-fcbd-459f-ae0f-2def574fb6d2)

Reply:
Ah, ok, thanks, I've misinterpreted -10000. Looks fine.